packages/mc-scripts/package.json

```diff
@@ -70,6 +72,7 @@
     "@vitejs/plugin-react-swc": "3.7.0",
     "autoprefixer": "^10.4.15",
     "babel-loader": "8.3.0",
+    "babel-plugin-formatjs": "^10.5.25",
```

Comment:
In Vite we are not using our custom babel preset but the `@vitejs/plugin-react`. These deps (used by the react babel plugin) should be listed explicitly.